### DISCUSSION TITLE
(50th, 75th, 95th, 99th) pct & Max Response Time

### DISCUSSION THREAD

**Discussion by maojunxyz** [naver/ngrinder#916] [Q&A] [answered]
Can the above statistics be added to the detailed statistics?

**Comment by donggyu04** [comment, marked as answer]
@maojunxyz You can add [User defined statistic](https://github.com/naver/ngrinder/wiki/Multiple-User-Defined-Statistic-In-nGrinder) in your script. if this doesn't meet your needs, there's no way.
